CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file, which follo
 on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 [Semantic Versioning](http://semver.org/).
 
+## [1.13.0] - 2017-03-02
```

Comment:
This should be ## [Unreleased] and then change to 1.13.0 on release.  The correct date should be put in at release time